src/components/Searcher.js

```diff
@@ -515,14 +521,15 @@ class Searcher extends React.Component<SearcherDefaultProps, SearcherProps, Sear
     if (createNewIfNotExisting) {
       const savedUser = AuthUtils.getSavedUser();
       const defaultSignalData = new SignalData();
-      const { query, queryTimestamp, relevancyModels } = this.state;
+      const { query, queryTimestamp } = this.state;
+      const relevancyModels = this.getRelevancyModels();
       if (savedUser) {
         defaultSignalData.locale = 'en';
         defaultSignalData.principal = `${AuthUtils.config.ALL.defaultRealm}:${savedUser.fullName}:${savedUser.userId}`;
         defaultSignalData.query = query;
         defaultSignalData.queryTimestamp = queryTimestamp;
         defaultSignalData.relevancyModelName = relevancyModels[0] || 'default';
-        defaultSignalData.relevancyModelNames = relevancyModels && relevancyModels.length > 0 ? relevancyModels : ['default'];
+        defaultSignalData.relevancyModelNames = relevancyModels.length > 0 ? relevancyModels : ['default'];
```

Comment:
Thinking about this, I'm not sure that we want to default it to a model called "default." I think we should leave it out altogether if we don't have anything specific to put here. (I.e., set `relevancyModelName` to `''` and `relevancyModelNames` to `[]`.)